<robot name="paper_box">
  <link name="paper_box">
  <static>false</static>
    <inertial>
      <origin xyz="0.15 0.1 0.1" />
      <mass value="0.5" />
      <inertia  ixx="0.0033" ixy="0.0"  ixz="0.0"  iyy=".00542"  iyz="0.0"  izz="0.00542" />
    </inertial>
    <visual>
      <origin xyz="0.15 0.1 0.1" />
      <geometry>
        <box size="0.3 0.2 0.2" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.15 0.1 0.1" />
      <geometry>
        <box size="0.3 0.2 0.2" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="paper_box">
    <material>Gazebo/Yellow</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>

<surface>
      <friction>
        <ode>
          <mu>10.0</mu>
          <mu2>10.0</mu2>
          <slip1>10</slip1>
          <slip2>10</slip2>
        </ode>
    <bullet>
    <friction>10.0</friction>
    <friction2>10.0</friction2>
    <fdir1>1 1 0</fdir1>
    <rolling_friction>1.0</rolling_friction>
    </bullet>
      </friction>

  <contact>
    <ode>
        <soft_cfm>0</soft_cfm>
        <soft_erp>0.1</soft_erp>
        <kp>1e+10</kp>      
        <kd>10</kd>
        <max_vel>0.01</max_vel>
        <min_depth>1</min_depth>
    </ode>
  </contact>
  </surface>
</robot>
